plugins/CoreAdminHome/templates/home.twig

```diff
@@ -21,9 +21,11 @@
             <div class="col s12 {% if isFeedbackEnabled %}m4{% else %}m6{% endif %}">
                 <div piwik-widget-loader='{"module":"CoreHome","action":"getSystemSummary"}'></div>
             </div>
-            <div class="col s12 {% if isFeedbackEnabled %}m4{% else %}m6{% endif %}">
-                <div piwik-widget-loader='{"module":"Installation","action":"getSystemCheck"}'></div>
-            </div>
+            {% if hasDiagnostics %}
```

Comment:
Can be also seen on http://builds-artifacts.piwik.org/piwik/piwik/widgetsadmin/22192/UIIntegrationTest_admin_home.png but will merge for now